### PR TITLE
DOC: Recommend adding dimension to switch between row and column vectors

### PR DESCRIPTION
**Pull request by iansan5653**
Relevant post on Stack Overflow: https://stackoverflow.com/questions/53303979/transpose-a-1-dimensional-array-in-numpy-without-casting-to-matrix

### CODE REVIEW

**Comment by eric-wieser** [comment]
Perhaps something like:

> To convert a 1d array into a 2d column vector, an extra dimension must be added. `np.atleast2d(a).T` achieves this, as does `a[:, np.newaxis]`

**Comment by seberg** [comment]
@iansan5653 could you rephrase it more like Eric suggested?

**Comment by seberg** [comment]
Reads fine to me thanks @iansan5653.
